src/mqtt_client/AtlasMqttClient.h

```diff
@@ -0,0 +1,28 @@
+#ifndef __ATLAS_MQTT_CLIENT_H__
+#define __ATLAS_MQTT_CLIENT_H__
+
+#include <stdlib.h>
+#include <string>
+
+using namespace std;
+
+namespace Atlas
+{
+class AtlasMqttClient
+{
+public:
+	static AtlasMqttClient& getInstance();
+    static void destroyInstance();
+
+	void publishMessage(const string address, const string clientID, const string topic, 
```

Comment:
please use std::string, and remove using namespace std

Reply:
Done.